dod.md

```diff
@@ -0,0 +1,75 @@
+# Pull request conventions (Definition of Done)
+
+## Project specific requirements
+
+This can vary from line-length, complexity, browser testing checklist and application environment support for example.
+Project team should decide themselves what kind of `definition of done` they shall benefit from.
+Style guides and code quality rules are usually project specific, but general best practices should still always apply.
+Project issue specification and action flow should be decided by the product owner or project manager.
+
+## Minimum requirements
+
+- [ ] Produced code passes tests
+- [ ] Code builds without errors
+- [ ] Unit tests written and passing
+      * Unit tests should be:
+        * Clear and easy to understand
+        * Reliable that they fail if a bug is in the system
+        * Fast (slow tests will be a nuisance in numbers)
+- [ ] Code is documented
+      * When doing public API functionality context of the code is written
+      * Clarification, when code might be unclear.
+      * Clear and well named functionality does not need a comment
+- [ ] Variable names convey their use case
+- [ ] Produced code matches presumed functionality
+      * Matches specifications given in the issue
+      * Works in desired environments
+- [ ] Code quality
+      * Code follows best practices
+        * Usually specified by the language, framework or library itself.
```

Comment:
These I think belong to coding standards.

Reply:
"code follows the coding standard document [link]" ?